website/docs/cli/cloud/migrating.html.md

```diff
@@ -0,0 +1,119 @@
+2. **`remote` backend:** If the working directory was already connected to Terraform Cloud with the `remote` backend, Terraform can continue using the same Terraform Cloud workspaces. The local names shown for those workspaces will change to match their remote names.
+3. **State backend or local state:** If the working directory already has state data in one or more workspaces (using either local state or a [state backend](/docs/language/settings/backends/index.html)), Terraform will try to migrate that state to new Terraform Cloud workspaces. You will need permission to manage workspaces in the destination Terraform Cloud organization, and you might need to rename your workspaces by adding a prefix and/or suffix. Terraform will interactively ask you what to do.
+
+The rest of this page briefly describes cases 2 and 3.
+
+## Migrating from the `remote` Backend
+
+The [`remote` backend](/docs/language/settings/backends/remote.html) was the primary
+implementation of Terraform Cloud's [CLI-driven run workflow](/docs/cloud/run/cli.html) for
+Terraform versions 0.11.13 through 1.0.x. We recommend using the native `cloud`
+integration for Terraform versions 1.1 or later, as it provides an improved user experience and
+various enhancements.
+
+### Block Replacement
+
+When switching from the `remote` backend to a `cloud` block, Terraform will try
+to continue using the same set of Terraform Cloud workspaces. To take advantage
+of this, you must replace your `backend "remote"` block with an equivalent
+`cloud` block:
+
+- If you were using a single workspace via the `name` argument, change the block
+  label to `cloud`. The inner arguments are unchanged.
+
+    ```diff
+     terraform {
+    -  backend "remote" {
+    +  cloud {
+         organization = "my-org"
+
+         workspaces {
+           name = "my-app-prod"
+         }
+       }
+     }
+    ```
+
+- If you were using multiple workspaces via the `prefix` argument, replace it
+  with a `cloud` block that uses the `tags` argument.
+
+    The existing workspaces don't need to already have these tags — when you
+    initialize, Terraform will attempt to add the specified tags to them.
+
+    ```diff
+     terraform {
+    -  backend "remote" {
+    +  cloud {
+         organization = "my-org"
+
+         workspaces {
+    -      prefix = "my-app-"
+    +      tags = ["app:mine"]
+         }
+       }
+     }
+    ```
+
+### Limitations
+
+To take advantage of automatic tagging of existing workspaces, you must have
+permission to change tags for every affected workspace. Thus, switching to the
+`cloud` integration works best when someone with permission to manage workspaces
+for your organization performs the migration first. When other users
+re-initialize later, the workspaces will already be tagged appropriately and
+tagging permissions won't be required.
```

Comment:
```suggestion
they will not need tagging permissions.
```

Reply:
Ehh, this section reads oddly to me. 

The overlying theme is actually that to effectively use the familiar CLI workflow with workspaces, you _need_ Manage Workspaces permission in Terraform Cloud. It's not a limitation so much as a requirement for the whole feature. You can still _somewhat_ use multiple workspaces if you do not have admin permission to workspaces, but that seems like a caveat that belongs more in TFC documentation elsewhere - a specialized workflow, one could say.